src/helpers/chartOptions.ts

```diff
@@ -1,50 +1,83 @@
 import type { ColorDefinition } from "./colors"
+import { ReactElement } from "react";
 
 export interface LineChartOptions {
-    lineColor?: string
-    domainMin?: number | "Auto"
+    lineColor?: string;
+    domainMin?: number | "Auto";
 }
 
 export interface BarChartOptions {
-    barColor?: ColorDefinition
-    thresholds?: BarChartThreshold[]
+    barColor?: ColorDefinition;
+    thresholds?: BarChartThreshold[];
 }
 
 export interface BarChartThreshold {
-    value: number
-    referenceLineColor?: ColorDefinition
-    overThresholdBarColor?: ColorDefinition
+    value: number;
+    referenceLineColor?: ColorDefinition;
+    overThresholdBarColor?: ColorDefinition;
 }
 
 export interface AreaChartOptions {
-    lineColor?: ColorDefinition
-    areaColor?: ColorDefinition
+    lineColor?: ColorDefinition;
+    areaColor?: ColorDefinition;
 }
 
 export interface ChartSeries {
-    dataKey: string
-    color?: ColorDefinition
+    dataKey: string;
+    color?: ColorDefinition;
 }
 
 export interface AreaChartSeries extends ChartSeries {
-    areaColor?: ColorDefinition
+    areaColor?: ColorDefinition;
 }
 
-export interface MultiSeriesLineChartOptions {
-    connectNulls?: boolean
-    hideDots?: boolean
-    thresholds?: ChartThreshold[]
-    domainMin?: number | "Auto"
-    domainMax?: number
+export interface TimeSeriesChartContainerOptions {
+    height?: string | number;
 }
 
-export interface MultiSeriesBarChartOptions {
-    thresholds?: BarChartThreshold[]
+export interface TimeSeriesChartXAxisOptions {
+    domain?: [number, number];
+    ticks?: (string | number)[];
+    tickFormatter?: (value: any, index: number) => string;
+}
+
+export interface TimeSeriesChartYAxisOptions {
+    width?: number;
+    domain?: [number, number];
```

Comment:
It's a little confusing to have this here as well as on the `MultiSeriesLineChartOptions`.  I realize you want it here because you want to be able to customize it for all charts and not just lines.  As you said, there's a few other "breaking" changes in this that I agree are not worth worrying about yet - so I'd probably advocate for removing `domainMin` and `domainMax` on `MultiSeriesLineChartOptions` and updating the handful of places internally that will break (which will likely be mostly a bunch of stories may or may not actually break at compile time).

Reply:
Ah, yeah, good call.  I'll do that as well.

Reply:
This change has been made.  I believe I have found/updated all of the existing usage of those two fields.